src/validator.spec.ts

```diff
@@ -1,82 +1,316 @@
-import { validator, setErrorMessage } from './validator';
+import { setErrorMessage, validator } from './validator';
 
-// TODO: Add specs
 describe('fonk-later-date-validator specs', () => {
+  it('should return succeeded validation when value is a valid Date object later than actual Date', () => {
+    const value = new Date(2020, 11, 24, 10, 33, 30, 0);
+    const date = new Date();
```

Comment:
This spec will fail next year xD. If you want a spec with current date, you could assign date + one day or something like that to `value`